lib/src/front_end/delimited_list_builder.dart

```diff
@@ -3,8 +3,8 @@
 // BSD-style license that can be found in the LICENSE file.
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/token.dart';
-import 'package:dart_style/src/ast_extensions.dart';
```

Comment:
Nice catch. I wish VS Code didn't automatically use package imports when it auto-adds an import from the same package.

Reply:
agreed!